packages/shared/src/components/cards/common/PostMetadata.tsx

```diff
@@ -49,6 +67,35 @@ export default function PostMetadata({
           <Separator />
         )}
         {!!description && description}
```

Comment:
If description is empty/null/false it should not show. We could remove the `!!description && ` if we don't expect numbers here
```suggestion
        {description}
```

Reply:
This is not a change made in this PR